app/api/dao/user.py

```diff
@@ -259,11 +259,11 @@ def update_user_profile(user_id: int, data: Dict[str, str]):
             else:
                 user.occupation = None
 
-        if "organization" in data:
-            if data["organization"]:
-                user.organization = data["organization"]
+        if "current_organization" in data:
```

Comment:
that's right, @devkapilbansal . because bit schema has an attribute called organization as well. So, I renamed it here on MS coz it clashes with the BIT attribute otherwise.